### ISSUE TITLE
Mods Issue

### ISSUE THREAD

**Issue by giogimic** [FezVrasta/ark-server-tools#200]
i have is all the mods i have are in game but no one can learn the engrams for them any of them.

**Comment by giogimic** [comment]
I Have the mods and .mod files inside "/ARK/ShooterGame/Content/Mods"

Inside my "ARK/ShooterGame/Saved/Config/LinuxServer" inside Game.ini have

[ServerSettings]
ActiveMods=520879363,510590313,509793590,507648142,505861851,510816108,520373758

I'm using ArkManager so inside of " /etc/arkmanager/arkmanager.cfg" 

i have

ark_GameModIds="520879363,510590313,509793590,520055042,507648142,505861851,510816108,520373758"

**Comment by klightspeed** [comment]
Is that `Game.ini` or `GameUserSettings.ini`?  `ActiveMods` belong in `GameUserSettings.ini`.

I am guessing that the people who are trying to learn the engrams have enough engram points and the required prerequisites to learn those engrams?

**Comment by giogimic** [comment]
Well i tried it in Game.ini when that didn't work i tried it in GameUserSettigns.ini  still a no go but ah yeah haha maxed my level and tested that way

**Comment by giogimic** [comment]
I Tried removing all the mods except 1 and still wont let me learn.

When i connect to the server it shows the loading/downloading mod screen once that is done it goes in and The HUD Mods won't display and the item mods i cant learn even with just 1 mod in there.

**Comment by FezVrasta** [comment]
Does it happens even without ARK Server Tools? I suspect it to be a problem of the mod itself

**Comment by giogimic** [comment]
Maybe, But i've tried a few different mods off that list above same issue ill try without arkmanager here in a bit.

**Comment by giogimic** [comment]
Found the issue, I was using steamcmd to download the mod with workshop_download_item but when i do that it for some reason doesn't download all the files? i had to upload the remaining files from my computer to get it to work. Which im still doing and will be doing the rest of the night because my internet sucks lol.

Sorry for wasting everyones time.

**Comment by klightspeed** [comment]
Were you trying to copy directly from `$steamcmdroot/steamapps/workshop/content/346110/$modid` to `$arkserverroot/ShooterGame/Content/Mods/$modid`?

The .z files downloaded by steamcmd need to be extracted.  1.4-dev includes support for installing mods using steamcmd.

**Comment by giogimic** [comment]
So how do i extract them then?

**Comment by benzon** [comment]
Looks like the issue is still there, i updated/installed the mods true arkmanager update --update-mods and it failed nothing worked everything where partialy downloaded it looks like.

**Comment by klightspeed** [comment]
@benzon please create another ticket for this, including what version of arkmanager you're using (if you're using the latest, you should be able to use `arkmanager --version`), the mod(s) that are failing, and the output of `arkmanager update --update-mods`.

**Comment by giogimic** [comment]
Yeah.. Not sure why theres replies to this i thought i closed it please
open your own tickets for issues. Hate getting emails about it haha :P
